Jenkinsfile

```diff
@@ -0,0 +1,93 @@
+pipeline {
```

Comment:
can we add a `# vim: syntax=groovy` line here since it got renamed from `pipeline.groovy` in one of your commits  to `Jenkinsfile` in another ?

Reply:
Here's something even nicer: https://github.com/martinda/Jenkinsfile-vim-syntax. It does Groovy but also knows Jenkins-specific stuff.

(I use this through [vim-polyglot](https://github.com/sheerun/vim-polyglot) actually, which I highly recommend!).

Reply:
that looks nice.